homeassistant/components/generic_thermostat/climate.py

```diff
@@ -15,8 +15,14 @@
     HVAC_MODE_COOL,
     HVAC_MODE_HEAT,
     HVAC_MODE_OFF,
+    PRESET_ACTIVITY,
     PRESET_AWAY,
+    PRESET_BOOST,
+    PRESET_COMFORT,
+    PRESET_ECO,
+    PRESET_HOME,
```

Comment:
I haven't reviewed other implementations of the `climate` component but does it make sense?
It seems confusing to me to have the `PRESET_HOME` and `None` because I understand that they are the same use case.

Reply:
Completely different. Consider None like manual or hold. None is the only preset you can't set.